### ISSUE TITLE
net-analyzer/scapy-2.4.4_p20210227: scapy-2.4.4_p20210227.tar.gz VERIFY FAILED!

### ISSUE THREAD

**Issue by WGH-** [pentoo/pentoo-overlay#703]
gentoo-mirror/pentoo@1336c9592ae9adb3e725bdcde728d5b424ad70a0 (looks the same as current master here 83f23679cb6e9d592a343fce31e9147281ef831b)

```
[ebuild     U  ] net-analyzer/scapy-2.4.4_p20210227::pentoo [2.4.4::gentoo] PYTHON_TARGETS="python3_8 -python3_7 -python3_9" 3241 KiB

Total: 1 package (1 upgrade), Size of downloads: 3241 KiB

Would you like to merge these packages? [Yes/No]

>>> Verifying ebuild manifests

>>> Emerging (1 of 1) net-analyzer/scapy-2.4.4_p20210227::pentoo
>>> Downloading 'http://mirror.yandex.ru/gentoo-distfiles/distfiles/e7/scapy-2.4.4_p20210227.tar.gz'
--2021-03-03 15:54:16--  http://mirror.yandex.ru/gentoo-distfiles/distfiles/e7/scapy-2.4.4_p20210227.tar.gz
Resolving mirror.yandex.ru... 2a02:6b8::183, 213.180.204.183
Connecting to mirror.yandex.ru|2a02:6b8::183|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-03-03 15:54:17 ERROR 404: Not Found.

>>> Downloading 'https://github.com/secdev/scapy/archive/f0e3e4452ec63618b09bad30b459b7eba09ebd6b.tar.gz'
--2021-03-03 15:54:17--  https://github.com/secdev/scapy/archive/f0e3e4452ec63618b09bad30b459b7eba09ebd6b.tar.gz
Resolving github.com... 140.82.121.4
Connecting to github.com|140.82.121.4|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/secdev/scapy/tar.gz/f0e3e4452ec63618b09bad30b459b7eba09ebd6b [following]
--2021-03-03 15:54:17--  https://codeload.github.com/secdev/scapy/tar.gz/f0e3e4452ec63618b09bad30b459b7eba09ebd6b
Resolving codeload.github.com... 140.82.121.9
Connecting to codeload.github.com|140.82.121.9|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [application/x-gzip]
Saving to: ‘/var/cache/distfiles/scapy-2.4.4_p20210227.tar.gz.__download__’

/var/cache/distfile     [   <=>              ]   3.16M  7.07MB/s    in 0.4s

2021-03-03 15:54:18 (7.07 MB/s) - ‘/var/cache/distfiles/scapy-2.4.4_p20210227.tar.gz.__download__’ saved [3318442]

!!! Fetched file: scapy-2.4.4_p20210227.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      3318442
!!! Expected: 3318375
Refetching... File renamed to '/var/cache/distfiles/scapy-2.4.4_p20210227.tar.gz._checksum_failure_.xfjo7lgm'

!!! Couldn't download 'scapy-2.4.4_p20210227.tar.gz'. Aborting.
 * Fetch failed for 'net-analyzer/scapy-2.4.4_p20210227', Log file:
 *  '/var/tmp/portage/net-analyzer/scapy-2.4.4_p20210227/temp/build.log'

>>> Failed to emerge net-analyzer/scapy-2.4.4_p20210227, Log file:

>>>  '/var/tmp/portage/net-analyzer/scapy-2.4.4_p20210227/temp/build.log'

 * Messages for package net-analyzer/scapy-2.4.4_p20210227:

 * Fetch failed for 'net-analyzer/scapy-2.4.4_p20210227', Log file:
 *  '/var/tmp/portage/net-analyzer/scapy-2.4.4_p20210227/temp/build.log'
 ```

**Comment by ZeroChaos-** [comment]
Does it fail every time with the same file size? It is odd that a commit tarball changes...

**Comment by WGH-** [comment]
Yes, it's stable for me.

**Comment by blshkv** [comment]
It happened exactly the same with a previous snapshot. No idea what's going on

**Comment by blshkv** [comment]
Doh, I found this is the old bug:
https://github.com/pentoo/pentoo-overlay/issues/287

**Comment by blshkv** [comment]
fixed in f7a2169cc1c926e77841f8cb6f6bbe77a90d91bf

**Comment by WGH-** [comment]
Wow, I didn't know Git could substitute things on archive export.
